lib/repomd_parser/repomd_xml_parser.rb

```diff
@@ -42,7 +42,7 @@ def parse
         checksum_type: hash[:checksum][:type],
         checksum: hash[:checksum][:value],
         type: type,
-        size: hash[:size][:value].to_i
+        size: hash[:size] ? hash[:size][:value].to_i : 0
```

Comment:
I've though about this for a while, perhaps `nil` would be better than 0:
* 0 size indicates that the file is empty (in theory, a repo can contain empty files);
* `nil` indicates that the size attribute is missing, which is the case with old-style repos.

What do you think, @srinidhibs?

Reply:
Sounds good @ikapelyukhin! Will push the changes in a bit.